src/test/java/hudson/maven/MojoInfoBuilder.java

```diff
@@ -44,12 +57,22 @@ public MojoInfoBuilder version(String version) {
         return this;
     }
     
+    public MojoInfoBuilder executionId(String executionId) {
+        this.executionId = executionId;
+        return this;
+    }
+    
+    public MojoInfoBuilder evaluator(ExpressionEvaluator evaluator) {
+        this.evaluator = evaluator;
+        return this;
+    }
+    
     public MojoInfoBuilder startTime(long startTime) {
         this.startTime = startTime;
         return this;
     }
     
-    public MojoInfoBuilder configValue(String key,String value) {
+    public MojoInfoBuilder configValue(String key, String value) {
```

Comment:
please avoid unrelated changes. they may impact merges due to parallel notifications